### ISSUE TITLE
[Using] Disable autocomplete and user's guide

### ISSUE THREAD

**Issue by Henrique1792** [brave/browser-laptop#3631]
Good evening everyone =)

I'm a rookie using brave, and I'd like to disable autocomplete. It's a preference that I have, you know, and I didn't found anything to help me find commands and etc.

Thanks for the support =)

**Comment by svnpenn** [comment]
Yes please. As a potential expat from Firefox, this is stopping me. Firefox has this option to disable it:

    browser.urlbar.autoFill

Brave can toggle "Show history matches", but that disables autocomplete and removes history matches from the dropdown as well

**Comment by svnpenn** [comment]
https://github.com/iridium-browser/tracker/issues/190

**Comment by svnpenn** [comment]
https://github.com/Eloston/ungoogled-chromium/issues/314

**Comment by svnpenn** [comment]
https://github.com/minbrowser/min/issues/555

**Comment by cup** [comment]
http://github.com/brave/brave-browser/issues/843
http://github.com/brave/browser-laptop/issues/3631
http://github.com/brave/browser-laptop/issues/7628
